docs/source/model_doc/vit.mdx

```diff
@@ -65,21 +65,23 @@ Tips:
 
 Following the original Vision Transformer, some follow-up works have been made:
 
-- DeiT (Data-efficient Image Transformers) by Facebook AI. DeiT models are distilled vision transformers. Refer to
-  [DeiT's documentation page](deit). The authors of DeiT also released more efficiently trained ViT models, which
-  you can directly plug into [`ViTModel`] or [`ViTForImageClassification`]. There
-  are 4 variants available (in 3 different sizes): *facebook/deit-tiny-patch16-224*, *facebook/deit-small-patch16-224*,
-  *facebook/deit-base-patch16-224* and *facebook/deit-base-patch16-384*. Note that one should use
-  [`DeiTFeatureExtractor`] in order to prepare images for the model.
-
-- BEiT (BERT pre-training of Image Transformers) by Microsoft Research. BEiT models outperform supervised pre-trained
+- [DeiT](./model_doc/deit) (Data-efficient Image Transformers) by Facebook AI. DeiT models are distilled vision transformers.
```

Comment:
@sgugger is this the correct relative path?

Reply:
No, good catch. This file is in `model_doc`, so you should just point to deit:
```suggestion
- [DeiT](deit) (Data-efficient Image Transformers) by Facebook AI. DeiT models are distilled vision transformers.
```